src/components/pages/NewProject/components/Description.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+function Description(props) {
+    return (
+        <div className="col">
+            <label>Description</label>
+            <input
```

Comment:
Change this to a textarea element so you can make the input field bigger

Reply:
update completed

Reply:
Change the *element* to textarea. See new risk page:
![image](https://user-images.githubusercontent.com/50573262/92359226-d381dd80-f0e2-11ea-914e-7b0fafadb5d7.png)

Reply:
done